utils/generate_types/index.js

```diff
@@ -32,7 +32,7 @@ let documentation;
   if (!fs.existsSync(typesDir))
     fs.mkdirSync(typesDir)
   fs.writeFileSync(path.join(typesDir, 'protocol.d.ts'), fs.readFileSync(path.join(PROJECT_DIR, 'src', 'server', 'chromium', 'protocol.ts')), 'utf8');
-  const browser = await chromium.launch();
+  const browser = await chromium.launch({ chromiumSandbox:false });
```

Comment:
why's this?

Reply:
Inside the Docker container we run the tests and generate types due install as pwuser which is not able to use the Chromium Sandbox.